src/modules/structure.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] To stick to the best practices, Kindly make sure you don't disable your linters rules as they help you keep your codebase clean and conform to the standards. Please make sure you delete that.
`The no-cycle error means simply you have some code you have exported from here into the index.js and then you have also some other methods you have imported from index.js. Which is not supposed to be the case`